orb/core/model.py

```diff
@@ -464,15 +449,17 @@ def id(self, **context):
         return self.get(self.schema().idColumn(), useMethod=False, **context)
 
     def init(self):
-        for column in self.schema().columns().values():
-            if not column.testFlag(column.Flags.Virtual):
-                value = column.default()
-                if column.testFlag(column.Flags.I18n):
-                    value = {self.__context.locale: value}
-                elif column.testFlag(column.Flags.Polymorphic):
-                    value = type(self).__name__
+        columns = self.schema().columns().values()
+        with WriteLocker(self.__dataLock):
```

Comment:
we were running into issues with the system getting locked up when we switched to gevent vs. threading.